src/Access/Authentication.h

```diff
@@ -103,7 +104,11 @@ class Authentication
 
     Type type = Type::NO_PASSWORD;
     Digest password_hash;
+
+    // Used and maintained only for LDAP.
```

Comment:
Makes sense to prefix them with `ldap` to avoid accidental errors, e.g. `ldap_last_successful_auth_timestamp`.

Reply:
I can do that.

Reply:
Committed the changes to names, for all but `server_name` - it is already renamed in a separate PR #14995, and it will cause nasty conflicts if I duplicate it here.